transport-protocol/.gitignore

```diff
@@ -1 +1,2 @@
+src/transport_protocol.rs
```

Comment:
I think this file is no longer created so it shouldn't be necessary to ignore it.